src/middlewares/requiredFieldsMiddleware.ts

```diff
@@ -85,13 +86,15 @@ const requiredFields = ({ getState, dispatch }: MiddlewareAPI<Dispatch<AnyAction
  * @param operationType Key of operation to check
  * @param actions List of operations and/or operation groups
  */
-function operationRequiresAutosave(operationType: string, actions: Array<Operation | OperationGroup>) {
+export function operationRequiresAutosave(operationType: string, actions: Array<Operation | OperationGroup>) {
     let result = false
     if (!actions) {
         console.error('rowMeta is missing in the middle of "sendOperation" action')
         return result
     }
-    result = flattenOperations(actions).some(action => action.type === operationType && action.autoSaveBefore)
+    result = flattenOperations(actions).some(
+        action => action.type === operationType && action.autoSaveBefore && action.type !== OperationTypeCrud.save
```

Comment:
Ideally we should never check for `OperationTypeCrud.save` directly; a helper might be useful that check both `action.type` and `action.actionRole`.

Reply:
> Ideally we should never check for `OperationTypeCrud.save` directly; a helper might be useful that checks both `action.type` and `action.actionRole`.

It seems to be unnecessary to create another one utility like `matchOperationRole`.
Because exactly in this case we already iterate over `flattenOperations`.
I guess this condition could be resolved as followed here:
```typescript
result = flattenOperations(actions).some(
        action =>
            action.type === operationType &&
            action.autoSaveBefore &&
            action.type !== OperationTypeCrud.save &&
            action?.actionRole !== OperationTypeCrud.save
     
    )
```

Reply:
checkOperationRole helper is added